src/odemis/gui/comp/overlay/world.py

```diff
@@ -899,7 +899,7 @@ def _draw_points(self, ctx):
 
                 blit = horz_dc.Blit
                 for i in range(buf_rep_x):
-                    x = i * step_x + half_step_x
+                    x = round(i * step_x + half_step_x)
```

Comment:
int would be nicer as this code is expecting int and the test cases would be expecting int as well. Round can lead to error in test cases because of it odd and even behavior.